modules/cco-short-term-creds-auth-flow-aws.adoc

```diff
@@ -6,6 +6,8 @@
 [id="cco-short-term-creds-auth-flow-aws_{context}"]
 = AWS Security Token Service authentication process
 
+Requests for new and refreshed credentials are automated by using an appropriately configured AWS IAM OpenID Connect (OIDC) identity provider, combined with AWS IAM roles. {product-title} signs service account tokens that are trusted by AWS IAM, and can be projected into a pod and used for authentication. Tokens are refreshed after one hour.
```

Comment:
```suggestion
Requests for new and refreshed credentials are automated by using an appropriately configured AWS IAM OpenID Connect (OIDC) identity provider combined with AWS IAM roles. Service account tokens that are trusted by AWS IAM are signed by {product-title} and can be projected into a pod and used for authentication. Tokens are refreshed after one hour.
```

Reply:
Thanks! I'm gonna go ahead and open another PR to get this editorial feedback into pre-4.14 (it is not gonna want to cherrypick :sweat_smile:)